hub/apps/get-started/index.md

```diff
@@ -93,6 +93,30 @@ For more information about Windows Forms apps, see the following articles:
 * [API reference (.NET)](/dotnet/api/index)
 * [Enhancing Windows Forms apps](/dotnet/framework/winforms/advanced/)
 
+#### [React Native for Windows](#tab/rnw)
+[React Native](https://reactnative.dev) is a development platform from Facebook which allows building cross-platform apps.
+[React Native for Windows](https://aka.ms/reactnative) brings React Native support for the Windows 10 and Windows 11 SDKs, enabling you to use JavaScript to build native Windows apps for all devices supported by Windows 10 and Windows 11, including PCs, tablets, 2-in-1s, Xbox, Mixed reality devices, etc. 
+
+With React Native for Windows, you write most or all of your app code in JavaScript - or TypeScript - and the framework produces a native UWP XAML application. If your app needs to call a platform API, you can usually do so through one of the many [community modules](https://reactnative.directory), or if a module does not yet exist, you can easily [write a native module to expose it](https://aka.ms/RNW-NativeModules).
+
+Here are some reasons to choose React Native for Windows:
+- You want to share code across platforms as much as possible, or you have web properties that you want to share code with.
+- Improved developer productivity and inner loop, thanks to fast refresh.
+- Your app's fundamentals (performance, accessibility, internationalization) are as good as a native UWP app.
```

Comment:
I believe I had commented on this earlier but assuming a developer isn't familiar with the performance, accessibility, and internalization of UWP, it might help to point them to those articles. From a guidance perspective, will developers clearly know what "good" is and how to achieve it. 

This is mainly important if this is a requirement or prerequisite to using React Native for Windows. But I don't see this as a blocker.